JSDemos/Demos/Charts/APIDisplayATooltip/Angular/app/app.component.ts

```diff
@@ -28,19 +28,18 @@ export class AppComponent {
     this.populationData = service.getPopulationData();
   }
 
-  pointClick(e: any) {
-    const point = e.target;
+  pointClick({ target: point }: DxPieChartTypes.PointClickEvent) {
     point.showTooltip();
-    this.selectedRegion = point.argument;
+    this.selectedRegion = point.data.region;
```

Comment:
Seems like this change isn't straight connected to type rework, did you check the demo behaviour?